scripts/analyze.sh

```diff
@@ -248,6 +248,12 @@ fi
 
 if check_for_require fs || check_config_file ZJS_FS; then
     >&2 echo Using module: FS
+    OBJ_NAME=$(grep "require *( *['\"]fs['\"] *)*;" $SCRIPT | cut -d'=' -f1 | cut -d' ' -f2)
+
+    if grep "$OBJ_NAME\..*(" tests/test-fs-async.js | grep -q -v "Sync"; then
```

Comment:
This refers to a test that's maybe only on your machine? I suggest you do submit a simple sample at least that uses this because there doesn't seem to be one.

I haven't figured out what this does but hardcoding that test here probably isn't right?